### ISSUE TITLE
How to custom the changeset file name by the user?

### ISSUE THREAD

**Issue by fireairforce** [atlassian/changesets#630]
## Problem

As we well konwn, the changeset will generate a random markdown file, i just want to know how the user can set the filename, sometimes maybe i don't want to use the random file name, because in our monorepo project, there are too many changeset files...

## Proposed solution

Maybe Provide an option in option file to allow user set changeset file name.

**Comment by Andarist** [comment]
I don't think this is something of a high priority for now at the moment - I'm also unsure if there is a huge value in this. You can always just rename the generated file to achieve whatever effect you want.
